routes/users.js

```diff
@@ -536,6 +536,19 @@ router.delete('/user/:id', async function(req, res) {
             res.status(403).send({ message: 'User owns some web sites, please change owner first!' });
             return;
         }
+        let projects = await dbsrv.mongo_projects().find({ owner: uid }).toArray();
+        if (projects && projects.length > 0) {
+            res.status(403).send({ message: 'User owns some projects, please change owner first!' });
+            return;
+        }
+        const allprojects = user.projects ? user.projects : [];
+        for (let project_name of allprojects) {
+            const users_in_project = await dbsrv.mongo_users().find({ 'projects': project_name }).toArray();
```

Comment:
Might be better to use .count() instead of toArray, and just check the count value after

Reply:
(Need to check if it works. It works in mongo directly, but well..)